src/content/resources/architectural-overview.md

```diff
@@ -560,21 +579,23 @@ By contrast, Flutter minimizes those abstractions,
 bypassing the system UI widget libraries in favor
 of its own widget set. The Dart code that paints
 Flutter's visuals is compiled into native code,
-which uses Skia (or, in the future, Impeller) for rendering.
-Flutter also embeds its own copy of Skia as part of the engine,
+which uses Impeller  for rendering.
+Flutter also embeds its own copy of Impeller as part of the engine,
```

Comment:
This reads a little awkard to me, I think it was trying to say that the copy of Skia used by flutter was different than the system Skia on Android - but there is no system Impeller.

Maybe something like "Impeller is shipped along with the application allowing..."